SIT.Manager/Services/Install/InstallerService.cs

```diff
@@ -759,4 +783,16 @@ public void CopyEftSettings(string targetInstallDir)
             _logger.LogError(ex, "Failed to copy EFT settings.");
         }
     }
+    
+    public async Task InstallBsgLauncher()
+    {
+        string bsgLauncherPath = Path.Combine(_configService.Config.InstallPath, "BSGLauncher.exe");
+        if (File.Exists(bsgLauncherPath))
+        {
+            return;
+        }
+
+        string bsgLauncherUrl = "https://prod.escapefromtarkov.com/launcher/download";
+        await _fileService.DownloadFile("BsgLauncher.exe", _configService.Config.InstallPath, bsgLauncherUrl, null);
+    }
```

Comment:
Can find any references to this method but the general sentiment is that we should avoid touching BSGs stuff

Reply:
Just some lingering stuff to make the install process work just like lutris' (install the official launcher, log in, download EFT, close the launcher and then proceed with the normal SIT installation process). If you'd rather have people install EFT through lutris first and then having to install SIT, that would be possible, but in terms of UX that would kinda suck

Reply:
Yeah I understand it would suck to have them do that but as a project we like to avoid stepping on BSGs toes at all to avoid creating problems